launcher/modManager/cmodlistview_moc.cpp

```diff
@@ -417,6 +422,113 @@ void CModListView::disableModInfo()
 	ui->updateButton->setVisible(false);
 }
 
+auto CModListView::buttonEnabledState(QString modName, ModState & mod)
+{
+	struct result {
+		bool disableVisible; bool enableVisible; bool installVisible; bool uninstallVisible; bool updateVisible; bool directoryVisible; bool repositoryVisible;
+		bool disableEnabled; bool enableEnabled; bool installEnabled; bool uninstallEnabled; bool updateEnabled; bool directoryEnabled; bool repositoryEnabled;
+	} res;
```

Comment:
Don't worry, we don't have limit on used line endings
```suggestion
	struct result {
		bool disableVisible;
		bool enableVisible;
		bool installVisible;
		bool uninstallVisible;
		bool updateVisible;
		bool directoryVisible;
		bool repositoryVisible;
		bool disableEnabled;
		bool enableEnabled;
		bool installEnabled;
		bool uninstallEnabled;
		bool updateEnabled; 
		bool directoryEnabled;
		bool repositoryEnabled;
	} res;
```